{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2020-12-21",
    "AffectedVersionRange": "1.7.1",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "43377443",
        "Fixer": "16708391"
    },
    "Package": {
        "Registry": "pip",
        "Name": "pytorch",
        "URL": "https://pypi.org/project/torch",
        "Downloads": "2886"
    },
    "CWEs": [
        {
            "ID": "94",
            "Description": "Improper Control of Generation of Code ('Code Injection')"
        }
    ],
    "CVSS": {
        "Version": "3.1",
        "AV": "P",
        "AC": "H",
        "PR": "L",
        "UI": "N",
        "S": "U",
        "C": "L",
        "I": "L",
        "A": "L",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "3.8"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/pytorch/pytorch",
        "Codebase": [
            "C++"
        ],
        "Owner": "pytorch",
        "Name": "pytorch",
        "Forks": "12278",
        "Stars": "46206",
        "ForkName": "pytorch"
    },
    "Permalinks": [
        "https://cdn.discordapp.com/attachments/749019614352244777/786332580811243550/Screenshot_from_2020-12-10_01-55-24.png"
    ],
    "References": [
        {
            "Description": "",
            "URL": ""
        }
    ],
    "PrNumber": "1337",
    "FixSubmissionCount": 2
}